### ISSUE TITLE
Back to menu

### ISSUE THREAD

**Issue by mahotilo** [juek/AnchorNavigation#1]
It will be useful if we will have possibility to setup anchor for Anchor Navigation section. E.g. for "back to menu" link.

**Comment by juek** [comment]
There is quite some headroom for improvements, such as support for different nav styles and all the options that are currently only hardcoded.

> … anchor for Anchor Navigation section …

Agree. Hmm, I could just steal it from OnePageToolkit ;-)

FYI: I made the plugin for a current project with some extensive lists (like bibliographies), which need to be  divided into alphabetically ordered groups. In this case I'll get lots of [ A | B | C | … ] navs on a single page–one between every group.

![grafik](https://user-images.githubusercontent.com/704283/42591210-5c49d024-8546-11e8-8c78-c56bd0cb039f.png)

**Comment by mahotilo** [comment]
> ...lots of [ A | B | C | … ] navs on a single page–one between every group...

Indeed, the described use does not require anchors for menu. But a simpler scenario, such as inner navigation through large blocks of text (Contents), do needs this. So please "steal" it:)
